nginx/nginx.conf

```diff
@@ -0,0 +1,33 @@
+upstream lando {
```

Comment:
This nginx config is just for running the prototype in Docker, right?

Reply:
Yes that is correct, it's used in `compose.yaml`.